docs/UserGuide.adoc

```diff
@@ -127,20 +127,35 @@ Marks the attendance of the student with index number 1 as present.
 * `attendance 3 at/0` +
 Marks the attendance of the student with index number 3 as absent.
 
-=== Schedule timetables: `schedule`
+=== Schedule events: `schedule`
 
 Schedules an event for the user to keep track of. +
-Format: `schedule event/EVENT_NAME date/DATE start/TIME_START end/TIME_END descr/DESCRIPTION`
+Format: `schedule event/EVENT_NAME date/DATE start/TIME_START end/TIME_END [descr/DESCRIPTION]`
 
 ****
-* The event will be scheduled and reflected on the user's list of events
-* Invalid dates and times will not be scheduled e.g. `32-12-2018` will not schedule any event. Also, the year range is between 1600 and 9999
+* The event will be scheduled and reflected on the user's list of events.
```

Comment:
perhaps briefly mention the uniqueness criteria here, in case for some reason the user wants to create 2 identical events

Reply:
Added a new point